sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_joined_v1/query.sql

```diff
@@ -25,4 +33,3 @@ USING
   (submission_date, sample_id, client_id)
 WHERE
   cd.submission_date = @submission_date
-  AND cde.submission_date = @submission_date
```

Comment:
It is not guaranteed for an clients_daily event to exist, so this could be NULL. Filtering here on `submission_date` will result in some clients_daily entries being removed although we'd want to keep them.